charts/kube-ovn-v2/values.yaml

```
@@ -0,0 +1,719 @@
+# -- Global configuration.
+# @section -- Global parameters
+global:
+  registry:
+    address: docker.io/kubeovn
+    imagePullSecrets: []
+  images:
+    kubeovn:
+      repository: kube-ovn
+      dpdkRepository: kube-ovn-dpdk
+      vpcRepository: vpc-nat-gateway
+      tag: v1.14.0
+      support_arm: true
+      thirdparty: true
+
+# -- Image configuration.
+# @section -- Global parameters
+# @default -- "{}"
+image:
+  # -- Pull policy for all images.
+  # @section -- Global parameters
+  pullPolicy: IfNotPresent
+
+# -- Name override.
+# @section -- Global parameters
+nameOverride: ""
+# -- Full name override.
+# @section -- Global parameters
+fullnameOverride: ""
+
+# -- Namespace in which the CNI is deployed.
+# @section -- Global parameters
+namespace: kube-system
+
+# -- Comma-separated list of IPs for each master node.
+# @section -- Global parameters
+masterNodes: ""
+# -- Label used to auto-identify masters.
+# @section -- Global parameters
+masterNodesLabel: "kube-ovn/role=master"
```

```suggestion
masterNodesLabel:
  kube-ovn/role: master
```

Let's make this value a KV pairs.

---

Done. Each label given in the dictionnary is matched. So it's not a labelSelector, you don't need all the labels to match to be elected a master. But I think it's a better alternative.